orttraining/orttraining/python/ort_trainer.py

```diff
@@ -648,6 +648,10 @@ def _init_session(self):
                 *self.model_desc_.outputs_,
                 IODescription(get_all_gradients_finite_arg_name(self.session), [1], torch.bool)]
 
+        if self.state_dict_:
```

Comment:
Did you mean to be able to pass 'state_dict_' and 'strict_' to the ORTTrainer initializer?

Reply:
This did result in a missing attribute error. So it needs to be set to _None_ on ORTTrainer(...) initialization. But no it appears the intent was to cache desired state on a call to load_state_dict(...) without ONNX model. Later, when ONNX model is defined, it takes note of cached state and attempts load_state_dict(...) again.